AppleScript/AppleScript.sublime-syntax

```diff
@@ -12,16 +14,52 @@ file_extensions:
 first_line_match: |-
   (?xi:
     ^ \#! .* \bosascript(?!\s+-l\s+JavaScript)\b  # shebang
```

Comment:
Maybe introduce `{{shebang_language}}` variable and related shebang context to highlight language name?

related with: https://github.com/sublimehq/Packages/pull/4219